urwid/widget.py

```diff
@@ -1412,7 +1414,7 @@ def _normalize_to_caption(self, text):
             return text
         if tu:
             return text.encode('ascii') # follow python2's implicit conversion
-        return text.decode('ascii')
+        return text.decode(sys.stdin.encoding)
```

Comment:
See https://github.com/geier/khal/pull/224 for discussion, IMO the encoding issue should be handled in `keypress`. If you agree I could provide a patch.